homeassistant/components/tailwind/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set up your Tailwind garage door opener to integrate with Home Assistant.\n\nTo do so, you will need to get the local control key and IP address of your Tailwind device. For more details, see the description below the fields down below.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "token": "Local control key token"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Tailwind device. You can find the IP address by going into the Tailwind app and selecting your Tailwind device's cog icon. The IP address is shown in the **Device Info** section.",
+          "token": "To find local control key token, browse to the [Tailwind web portal]({url}), log in with your Tailwind account, and select the [**Local Control Key**]({url}) tab. The 6-digit number shown is your local control key token."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unsupported_firmware": "The firmware of your Tailwind device is not supported. Please update your Tailwind device to the latest firmware version using the Tailwind app."
+    }
+  },
+  "entity": {
+    "number": {
+      "brightness": {
+        "name": "Status LED"
+      }
+    }
```

Comment:
This would make it `number.tailwind_something_status_led` I think having brightness in the name would add value

Reply:
I've removed it, as it already implies. Otherwise it becomes unreadable on the device page by default.

Reply:
[f167420](https://github.com/home-assistant/core/pull/105926/commits/f167420159498a156fabb44992683c64297ca69a)